internal/spot/interfaces.go

```diff
@@ -108,6 +111,21 @@ type (
 		Build() (*OceanRollout, error)
 	}
 
+	// OceanSparkInterface defines the interface of the Spot Ocean for Apache Spark API.
+	OceanSparkInterface interface {
+		// ListClusters returns a list of Ocean Spark clusters.
+		ListClusters(ctx context.Context, controllerClusterID string, state string) ([]*OceanSparkCluster, error)
+
+		// GetCluster returns an Ocean Spark cluster spec by ID.
+		GetCluster(ctx context.Context, clusterID string) (*OceanSparkCluster, error)
```

Comment:
@thorsteinnth I'm confused.
ClusterID refers to controllerClusterID or OceanID?
Same for 'DeleteCluster' method

Reply:
This refers to the Ocean Spark cluster ID (osc-XXX)